_posts/en/newsletters/2024-07-26-newsletter.md

```diff
@@ -145,7 +145,59 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [Why is restructure of mempool required with cluster mempool?]({{bse}}123682)
+  Murch explains issues with Bitcoin Core’s current mempool data structure, how
+  cluster mempool addresses those issues, and how [cluster mempool][topic
+  cluster mempool] may be deployed in Bitcoin Core.
+
+- [DEFAULT_MAX_PEER_CONNECTIONS for Bitcoin core is 125 or 130?]({{bse}}123645)
+  Lightlike clarifies that while the maximum number of automatic peer
+  connections is 125 in Bitcoin Core, a node operator might add up to 8
+  additional connections manually.
+
+- [Why do protocol developers work on maximizing miner revenue?]({{bse}}123679)
+  David A. Harding lists several advantages of being able to predict which
+  transactions get into a block using the assumption that miners will maximize
+  fee revenue, noting “This allows spenders to make reasonable feerate
+  estimates, volunteer relay nodes to operate with a modest amount of bandwidth
+  and memory, and small decentralized miners to earn just as much fee revenue as
+  large miners”.
+
+- [Is there an economic incentive to use P2WSH over P2TR?]({{bse}}123500)
+  Vojtěch Strnad points out that while certain uses of P2WSH can be cheaper than
+  P2TR outputs, most use cases of P2WSH, like multisig and LN, would benefit
+  from the reduced fees enabled by hiding unused script paths in [taproot][topic
+  taproot] and using [schnorr signatures][topic schnorr signatures] for
+  key-aggregation schemes like [MuSig2][topic musig] and FROST.
+
+- [How many blocks per second can be sustainably be created using a time warp attack?]({{bse}}123698)
```

Comment:
NOTE, can't fix this here (yet), but I made an edit suggestion on StackExchange. 
```suggestion
- [How many blocks per second can sustainably be created using a time warp attack?]({{bse}}123698)
```

Reply:
Update, this is now fixed on SE, so can be fixed here too.